modules/renter/hostdb/persist.go

```diff
@@ -75,3 +71,27 @@ func (hdb *HostDB) load() error {
 	hdb.lastChange = data.LastChange
 	return nil
 }
+
+// threadedSaveLoop saves the hostdb to disk every 2 minutes, also saving when
+// given the shutdown signal.
+func (hdb *HostDB) threadedSaveLoop(shutChan chan struct{}) {
```

Comment:
heh the code I wrote is incorrect anyway, because the `Stop` function isn't blocked, it'll continue immediately after closing the channel and not wait for the save to happen.

I'll switch it to your method.